docs/SHAPCLI.md

```diff
@@ -0,0 +1,102 @@
+# SHAPCLI
+
+shapcli is an executable tool to use the api provided by shaptools. It wraps most of the commands and
+exposes them as a command line tool.
+
+In order to use the utility the `shaptools` library must be installed (either using `pip` or `rpm` package).
+
+## Motivation
+
+The major motivation behind this tools is to provide an easy way to run all of the tools provided by
+SAP regarding HANA. It wraps the next commands: `HDB`, `hdbnsutil`, `hdbuserstore`,
+`HDBSettings.sh`, `hdbsql`.
+
+Using this tool be avoid the need to change to SAP users every time we need to run any of these
+commands. This is really helpful when we are running other commands in the same time (as `crmsh`
+commands for example). Besides, having all of them gathered in the same place makes the usage
+easier.
+
+## How to use
+
+`shapcli` can be used providing the SAP HANA database information through command line or using a
+json configuration file (the options are mutually exclusive).
+Here an example of how to create the configuration file: [config.json](shapcli.config.example)
+
+Check how it works and help output running:
+
+```
+shapcli -h
+```
+
+This is the output:
+
+```
```

Comment:
having done some CLI in past, I'm not against to have the ouput of `shapcli -h` in README.md or docs, but the risk problem is that if we remove/add new things, this are potential places where we need to remember to update the doc. IMHO we could remove  and just refer to the `-h`

Reply:
Ok, agree. I will remove the outputs, just leaving the usage examples